sio.go

```diff
@@ -115,6 +115,22 @@ type Config struct {
 	PayloadSize int
 }
 
+// Overhead returns the number of bytes added to
+// a stream of a given size by DARE 2.0.
+//
+// The length of an encrypted stream is: size + Overhead(size)
+// Overhead panics if the size is larger than 2^48.
+func Overhead(size uint64) uint64 {
```

Comment:
We need to know without the overhead version as well, i.e to know what will be effective decrypted size given a size 'n'?

Reply:
I tried to keep it close the the `cipher.AEAD` [API](https://golang.org/pkg/crypto/cipher/#AEAD)
I guess for most users only the encrypted size is interesting.
However I think it is also valid to provide symmetric functions - esp. since we require it at the server. 
I make a proposal in the next commit version.